lib/palindrome_permutation.rb

```diff
@@ -1,5 +1,16 @@
 
 def palindrome_permutation?(string)
+  string_array = string.split("")
+  letters_hash = {}
 
- 
-end # method
+  string_array.each do |letter|
+    if letters_hash.keys.include?(letter)
```

Comment:
Using letters_hash.keys.include? converts the keys to an array, and then does an array lookup for the letter, which is O(n).  Try using letters_hash.has_key? to take advantage of hashtables O(1) lookup.